Doc/library/importlib.rst

```diff
@@ -258,28 +235,6 @@ ABC hierarchy::
                                      +-- SourceLoader
 
 
-.. class:: Finder
```

Comment:
On new lines 245 & 281, could you change:

```diff
- No longer a subclass of :class:`Finder`.
+ No longer a subclass of :class:`!Finder`.
```

this is needed as the cross-reference target has been removed here. GitHub won't let me do an in-line suggestion, sorry!